overrides/main.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block announce %}
+    <a class="announce-bar" href="/community_resources/access_workshop_2025/">
```

Comment:
```suggestion
    <a class="announce-bar" href="{{ base_url }}/community_resources/access_workshop_2025/">
```
`{{ base_url }}` is needed to keep the link for the PR previews, otherwise it would always link the main website (even from PR preview websites)

Reply:
An alternative would be to change the plugin hook from `on_page_content` to `on_post_page`, but I would need to test it so, for now, I will implement this workaround (because the banner is temporary anyway).

Reply:
Raised [an issue](https://github.com/ACCESS-NRI/mkdocs_resolve_absolute_urls_plugin/issues/1) about it.